jose/backends/cryptography_backend.py

```diff
@@ -20,7 +20,7 @@
 from cryptography.hazmat.primitives.keywrap import aes_key_wrap, aes_key_unwrap, InvalidUnwrap
 from cryptography.hazmat.primitives.padding import PKCS7
 from cryptography.hazmat.primitives.serialization import load_pem_private_key, load_pem_public_key
-from cryptography.utils import int_from_bytes, int_to_bytes
+from cryptography.utils import int_to_bytes
```

Comment:
If you can use int.from_bytes (it did not exist in 2.7), you can also use the to_bytes method of int.

Reply:
True, but interestingly enough cryptography didn't bother to deprecate that one.

Obviously this lifts the required python to 3.2+, but crypto itself has 3.6+ so this is necessary anyways.

Reply:
I am facing the same issue.